### ISSUE TITLE
ossec read log from graylog

### ISSUE THREAD

**Issue by GlobuleBleu** [ossec/ossec-hids#884]
HI, 

all my logs are send to graylog server. Is there a way to ossec read apache log (and all log) from remote graylog server?

**Comment by ddpbsd** [comment]
Nope. If graylog2 has an api to help with this, you could write something to pull the logs and dump them to a file.

**Comment by atomicturtle** [comment]
close no response
